cmd/bnsd/x/account/README.md

```diff
@@ -0,0 +1,21 @@
+# `account`
+
+
+Action | Has Superuser | Domain Admin | Account Owner | Anyone
+-- | -- | -- | -- | --
+Registar a domain | yes | no | no | yes
+Registar a domain | no | no | no | no
+Change domain admin | yes | yes | no | no
+Change domain admin | no | yes | no | no
```

Comment:
A domain without a superuser should not be able to be transferred.  (Confirmed by Antoine.)